server/graphql/common/expenses.ts

```diff
@@ -1533,11 +1568,34 @@ export const prepareExpenseItemInputs = async (
     );
   }
 
+  const mapItemUrlToUploadedFile: Record<string, string> = {};
```

Comment:
:ice_cream: I got confused expecting a `Record<string, UploadedFile>` when reading the code below

```suggestion
  const mapItemUrlToUploadedFileUrl: Record<string, string> = {};
```